### PR TITLE
Add `osu! Indonesia Presents: Ascaveth Invitational Tournament 2023` news post

### PR DESCRIPTION
**Pull request by Nivalyx**
Talked about this in #osu-news and with @Ephemeralis via PM. Scheduled to be released on Sunday if possible.

To do :

- [x] Banner image (will be coming tomorrow)
- [x] Video trailer (also will be coming tomorrow)

### CODE REVIEW

File: news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- ::{ flag=ID }:: [Hakui Koyori](https://osu.ppy.sh/users/10717635)
+- ::{ flag=ID }:: [Skydiver](https://osu.ppy.sh/users/4750008)
+- ::{ flag=ID }:: [lifeline](https://osu.ppy.sh/users/11367222)
+- ::{ flag=ID }:: [rHO](https://osu.ppy.sh/users/1629553)
+- ::{ flag=ID }:: [NazunaAmemiya](https://osu.ppy.sh/users/12159899)
+- ::{ flag=ID }:: [Fuma](https://osu.ppy.sh/users/1501956)
+- ::{ flag=ID }:: [ThatNOOBGuy](https://osu.ppy.sh/users/11091594)
+- ::{ flag=ID }:: [DEETO](https://osu.ppy.sh/users/10069909)
+
+The tournament is scheduled to commence at **Saturday, December 9th** on **01:00 UTC** (08:00 local time). Unfortunately however, due to logistical and some other concerns, the tournament isn't open for public attendance. But fear not! As you might already expect, the entirety of the event will be streamed live over at **[the osuIndonesia Twitch channel](https://www.twitch.tv/osuIndonesia)** as well as **[lifeline's Twitch channel](https://www.twitch.tv/lifeline)**.
```
Comment (0x84f) [inline news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md:28]:
```suggestion
The tournament is scheduled to commence on **Saturday, December 9th** on **01:00 UTC** (08:00 local time). Unfortunately however, due to logistical and some other concerns, the tournament isn't open for public attendance. But fear not! As you might already expect, the entirety of the event will be streamed live over at **[the osuIndonesia Twitch channel](https://www.twitch.tv/osuIndonesia)** as well as **[lifeline's Twitch channel](https://www.twitch.tv/lifeline)**.
```

File: news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/COMINGSOON" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Merchandise
+
+But wait, there's more!
+
+If you're in Indonesia, you're also able to support the tournament by purchasing exclusive merchandises ranging from keychains, stickers, to t-shirts —  all starring the official mascot of the event, **[Stella Eirua](https://twitter.com/AscavethInv/status/1723545814449160621)**.
```
Comment (0x84f) [inline news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md:40]:
```suggestion
If you're in Indonesia, you're also able to support the tournament by purchasing exclusive merchandises ranging from keychains, stickers, to t-shirts — all starring the official mascot of the event, **[Stella Eirua](https://twitter.com/AscavethInv/status/1723545814449160621)**.
```

File: news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+You can place your order [here](https://docs.google.com/forms/d/e/1FAIpQLScmKN4kAXtiIluLchGfeodKpx37tcHd6OIe_6w1ZfLelDJWgA/viewform), or you can also check out the following short trailer for the merchandise line:
```
Comment (Walavouchey) [inline news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md:42]: this kind of thing is quite unprecedented for news posts but will see what others think

Comment (Nivalyx) [reply news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md]: yeah i'm aware that this is not something that's commonly included in a news post promoting a community event and i'm not 100% sure whether to have this section or not either  ¯\_(ツ)_/¯ 

but yes, i suppose the inclusion is worth a try (on the pr at least) and from there we can see what others think about this

File: news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (peppy) [reply news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md]: I think it's alright as they are selling their own original merchandise as a limited run.

**Review by peppy — APPROVED** [review]

File: news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/COMINGSOON" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Merchandise
+
+But wait, there's more!
+
+If you're in Indonesia, you're also able to support the tournament by purchasing exclusive merchandise ranging from keychains and stickers to t-shirts —  all starring the official mascot of the event, **[Stella Eirua](https://twitter.com/AscavethInv/status/1723545814449160621)**.
+
+Check out the following short trailer for the merchandise line below and, if this strikes you fance, you can place your order [here](https://docs.google.com/forms/d/e/1FAIpQLScmKN4kAXtiIluLchGfeodKpx37tcHd6OIe_6w1ZfLelDJWgA/viewform):
```
Comment (0x84f) [inline news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md:42]:
```suggestion
Check out the following short trailer for the merchandise line below and, if this strikes you fancy, you can place your order [here](https://docs.google.com/forms/d/e/1FAIpQLScmKN4kAXtiIluLchGfeodKpx37tcHd6OIe_6w1ZfLelDJWgA/viewform):
```

File: news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/COMINGSOON" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Merchandise
+
+But wait, there's more!
+
+If you're in Indonesia, you're also able to support the tournament by purchasing exclusive merchandise ranging from keychains and stickers to t-shirts —  all starring the official mascot of the event, **[Stella Eirua](https://twitter.com/AscavethInv/status/1723545814449160621)**.
```
Comment (0x84f) [inline news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md:40]:
```suggestion
If you're in Indonesia, you're also able to support the tournament by purchasing exclusive merchandise ranging from keychains and stickers to t-shirts — all starring the official mascot of the event, **[Stella Eirua](https://twitter.com/AscavethInv/status/1723545814449160621)**.
```

File: news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md (hunk 4 above)
Comment (Nivalyx) [reply news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md]: bruh what was i thinking when i made this typo lol

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**